extensions/blocks/anchor-fm/editor.js

```diff
@@ -53,32 +56,27 @@ async function insertSpotifyBadge() {
 }
 
 const ConvertToAudio = () => (
-	<PluginPostPublishPanel>
+	<PluginPostPublishPanel className="anchor-post-publish-outbound-link">
 		<p className="post-publish-panel__postpublish-subheader">
 			<strong>{ __( 'Convert to audio', 'jetpack' ) }</strong>
 		</p>
 		<p>{ __( 'Let your readers listen to your post.', 'jetpack' ) }</p>
 		<p>
 			<a href="https://anchor.fm/wordpress" target="_top">
 				{ __( 'Create a podcast episode', 'jetpack' ) }
-				<Icon icon={ external } className="components-external-link__icon" />
+				<Icon icon={ external } className="anchor-post-publish-outbound-link__external_icon" />
 			</a>
 		</p>
 	</PluginPostPublishPanel>
 );
 
 function showPostPublishOutboundLink() {
-	registerPlugin( 'post-publish-anchor-outbound-link', {
+	registerPlugin( 'anchor-post-publish-outbound-link', {
 		render: ConvertToAudio,
 	} );
 }
 
 function initAnchor() {
-	const isExtensionAvailable = getJetpackExtensionAvailability( name )?.available;
```

Comment:
As noted by @retrofox in https://github.com/Automattic/jetpack/pull/17936/files#r538196072, this is not needed since the extension does not rely on any plan check, and it's only loaded when the beta blocks are enabled.